src/Sterling/Account.php

```diff
@@ -20,8 +20,9 @@ public function InterbankTransferReq($data)
         try {
             $data['path'] = '/sterling/accountapi/api/Spay/InterbankTransferReq';
             $payload = isset($data['payload']) ? $data['payload'] : '';
+            $host = isset($data['host']) ? $data['host'] : '';
```

Comment:
What host are you setting to `""` here?

Reply:
the host is not compulsory so I am setting it to '' if it not provided.